neo4j_utils/_driver.py

```diff
@@ -830,18 +830,17 @@ def db_status(
 
         name = name or self.current_db
 
-        query = f'SHOW DATABASES WHERE name = "{name}";'
+        query = 'SHOW DATABASES'
 
         with self.fallback():
 
-            try:
-                resp, summary = self.query(query=query)
-            except neo4j.exceptions.ClientError:
-                return None
-
-        if resp:
+            resp, summary = self.query(query=query)
+            databases = [record['name'] for record in resp]
```

Comment:
why list the names of all the databases, when we're wondering about only one?

Reply:
Related to the comment above ("In the case that the database we are looking does not exist, this query fails.")